src/Angor/Client/Pages/Invest.razor

```diff
@@ -965,59 +970,68 @@ else
 
             var investorProject = (InvestorProject)project;
 
-            // remove signatures when requesting founder to sign
-            investorProject.SignaturesInfo = new()
-                {
-                    ProjectIdentifier = investorProject!.ProjectInfo.ProjectIdentifier,
-                };
-
-            var strippedInvestmentTransaction = network.CreateTransaction(investorProject.SignedTransactionHex);
-            strippedInvestmentTransaction.Inputs.ForEach(f => f.WitScript = Blockcore.Consensus.TransactionInfo.WitScript.Empty);
-
-            var accountInfo = storage.GetAccountInfo(network.Name);
-
-            var words = await passwordComponent.GetWalletAsync();
-
-            var investorNostrPrivateKey = _derivationOperations.DeriveProjectNostrPrivateKey(words, project.ProjectInfo.FounderKey);
-            var nostrPrivateKeyHex = Encoders.Hex.EncodeData(investorNostrPrivateKey.ToBytes());
-
-            var releaseAddress = accountInfo.GetNextReceiveAddress();
-
-            SignRecoveryRequest signRecoveryRequest = new()
-                {
-                    ProjectIdentifier = investorProject.ProjectInfo.ProjectIdentifier,
-                    InvestmentTransactionHex = strippedInvestmentTransaction.ToHex(network.Consensus.ConsensusFactory),
-                    UnfundedReleaseAddress = releaseAddress,
-                    AdditionalNpub = Investment.AdditionalNpub
-                };
-
-            var sigJson = serializer.Serialize(signRecoveryRequest);
-
-            var encryptedContent = await encryption.EncryptNostrContentAsync(
-                nostrPrivateKeyHex, investorProject.ProjectInfo.NostrPubKey,
-                sigJson);
-
-            var investmentSigsRequest = _SignService.RequestInvestmentSigs(encryptedContent, nostrPrivateKeyHex, investorProject.ProjectInfo.NostrPubKey);
-
-            investorProject.SignaturesInfo!.TimeOfSignatureRequest = investmentSigsRequest.eventTime;
-            investorProject.SignaturesInfo!.SignatureRequestEventId = investmentSigsRequest.eventId;
-            investorProject.InvestorNPub = NostrPrivateKey.FromHex(nostrPrivateKeyHex).DerivePublicKey().Hex;
-            investorProject.UnfundedReleaseAddress = releaseAddress;
-            investorProject.AdditionalNpub = Investment.AdditionalNpub;
-
-            storage.AddInvestmentProject(investorProject);
-
-            foreach (var input in strippedInvestmentTransaction.Inputs)
-                accountInfo.UtxoReservedForInvestment.Add(input.PrevOut.ToString());
+        investorProject.SignaturesInfo = new()
+        {
+            ProjectIdentifier = investorProject.ProjectInfo.ProjectIdentifier
+        };
 
-            storage.SetAccountInfo(network.Name, accountInfo);
+        var strippedTx = network
```

Comment:
I wonder why change the name of the param? is it too long? this just makes it harder to review the actual changes in the pr, pay attention to only change stuff related to the bug you are fixing.